File: cfgs/pythoncfgs/WRITE_ONE_LINE_C_FUNCTION_TO_FIND_WHETHER_A_NO_IS_POWER_OF_TWO.py
Nodes
1;conditional;foo_1 == 0;8;
2;conditional;foo_1 != 1;10;
3;statement;return true;14;
4;conditional;foo_1 % 2 != 0;11;
5;statement;foo_1 = foo_2 // 2;13;
6;statement;return false;12;
7;statement;return false;9;
8;exit;;
Edges
0;2;7;
1;3;4;
2;8;
3;5;6;
4;2;
5;8;
6;8;
7;

File: cfgs/java_cfgs_foo/WRITE_ONE_LINE_C_FUNCTION_TO_FIND_WHETHER_A_NO_IS_POWER_OF_TWO.java
Nodes
1;special;;;
2;conditional;foo_1 == 0;14;
3;statement;return false;14;
4;exit;;;
5;conditional;foo_1 != 1;15;
6;conditional;(foo_1 % 2) != 0;16;
7;statement;return false;16;
8;statement;n = foo_1 / 2;17;
9;statement;return true;19;
Edges
0;2;
1;3;5;
2;4;
3;
4;9;6;
5;7;8;
6;4;
7;5;
8;4;
